packages/threaddb/README.md

```diff
@@ -0,0 +1,21 @@
+# @textile/threaddb
```

Comment:
Using the minimal README here like others. A blog post will include all current README/docs. Also these docs should be added to the docs site.